Buildings/Controls/OBC/ASHRAE/G36/Atomic/SystemRequestsReheatBox.mo

```diff
@@ -0,0 +1,852 @@
+  CDL.Interfaces.RealInput uCoo(min=0, max=1, final unit="1")
+    "Cooling loop signal"
+    annotation (Placement(transformation(extent={{-220,70},{-180,110}}),
+      iconTransformation(extent={{-120,40},{-100,60}})));
+  CDL.Interfaces.RealInput VDisAir(
+    min=0,
+    final unit="m3/s",
+    quantity="VolumeFlowRate")
+    "Measured discharge airflow rate"
+    annotation (Placement(transformation(extent={{-220,-90},{-180,-50}}),
+      iconTransformation(extent={{-10,-10},{10,10}},rotation=0, origin={-110,0})));
+  CDL.Interfaces.RealInput VDisAirSet(
+    min=0,
+    final unit="m3/s",
+    quantity="VolumeFlowRate")
+    "Discharge airflow rate setpoint"
+    annotation (Placement(transformation(extent={{-220,10},{-180,50}}),
+      iconTransformation(extent={{-10,-10},{10,10}},rotation=0, origin={-110,20})));
+  CDL.Interfaces.RealInput uDam(
+    min=0, max=1, final unit="1") "Damper position"
+    annotation (Placement(transformation(extent={{-220,-170},{-180,-130}}),
+      iconTransformation(extent={{-10,-10},{10,10}},rotation=0, origin={-110,-20})));
+  CDL.Interfaces.RealInput TDisAirSet(
+    final unit="K",
+    quantity="ThermodynamicTemperature") if hotWatCoi
+    "Discharge airflow setpoint temperature for heating"
+    annotation (Placement(transformation(extent={{-220,-230},{-180,-190}}),
+      iconTransformation(extent={{-120,-60},{-100,-40}})));
+  CDL.Interfaces.RealInput TDisAir(
+    final unit="K",
+    quantity="ThermodynamicTemperature") if hotWatCoi
+    "Measured discharge airflow temperature"
+    annotation (Placement(transformation(extent={{-220,-310},{-180,-270}}),
+      iconTransformation(extent={{-120,-80},{-100,-60}})));
+  CDL.Interfaces.RealInput uHotVal(
+    min=0, max=1, final unit="1") if hotWatCoi
+    "Hot water valve position"
+    annotation (Placement(transformation(extent={{-220,-370},{-180,-330}}),
+      iconTransformation(extent={{-10,-10},{10,10}},rotation=0, origin={-110,-90})));
+  CDL.Interfaces.IntegerOutput yZonPreResReq
+    "Zone static pressure reset requests"
+    annotation (Placement(transformation(extent={{180,-50},{200,-30}}),
+      iconTransformation(extent={{100,-10},{120,10}})));
+  CDL.Interfaces.IntegerOutput yZonTemResReq
+    "Zone cooling supply air temperature reset request"
+    annotation (Placement(transformation(extent={{180,190},{200,210}}),
+      iconTransformation(extent={{100,60},{120,80}})));
+  CDL.Interfaces.IntegerOutput yHotValResReq if hotWatCoi
+    "Hot water reset requests"
+    annotation (Placement(transformation(extent={{180,-250},{200,-230}}),
+      iconTransformation(extent={{100,-60},{120,-40}})));
+  CDL.Interfaces.IntegerOutput yBoiPlaReq if (hotWatCoi and boiPla)
+    "Boiler plant request"
+    annotation (Placement(transformation(extent={{180,-440},{200,-420}}),
+      iconTransformation(extent={{100,-100},{120,-80}})));
+
+  CDL.Discrete.Sampler samTCooSet(
+    final startTime=0,
```

Comment:
startTime = 0 will disable the sampler if someone starts the simulation at a time smaller than zero, e.g., at startTime=86400.
We need to rewrite our sampler. I removed the startTime=0 assignment and will deal with it in https://github.com/lbl-srg/modelica-buildings/issues/938